spec/instant.html

```diff
@@ -561,13 +553,14 @@ <h1>AddInstant ( _epochNanoseconds_, _hours_, _minutes_, _seconds_, _millisecond
         The abstract operation AddInstant adds a duration in various time units to a number of nanoseconds since the Unix epoch.
       </p>
       <emu-alg>
-        1. Let _result_ be _epochNanoseconds_ + _nanoseconds_ +
-            _microseconds_ × 1000 +
-            _milliseconds_ × 1,000,000 +
-            _seconds_ × 1,000,000,000 +
-            _minutes_ × 60,000,000,000 +
-            _hours_ × 3,600,000,000,000.
-        1. Perform ? RejectInstant(_result_).
+        1. Assert: _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, and _nanoseconds_ are integer Number values.
+        1. Let _result_ be _epochNanoseconds_ + ℤ(_nanoseconds_) +
+            ℤ(_microseconds_) × *1000*<sub>ℤ</sub> +
+            ℤ(_milliseconds_) × *10<sup>6</sup>*<sub>ℤ</sub> +
+            ℤ(_seconds_) × *10<sup>9</sup>*<sub>ℤ</sub> +
+            ℤ(_minutes_) × *6 × 10<sup>10</sup>*<sub>ℤ</sub> +
+            ℤ(_hours_) × *3.6 × 10<sup>12</sup>*<sub>ℤ</sub>.
```

Comment:
```suggestion
            ℤ(_minutes_) × *6<sub>ℤ</sub> × 10<sup>10</sup>*<sub>ℤ</sub> +
            ℤ(_hours_) × *3.6<sub>ℤ</sub> × 10<sup>12</sup>*<sub>ℤ</sub>.
```

Reply:
ah right, duh. Either of your suggestions is fine; as long as every number is in the same space.

Reply:
I'll go with 3600<sub>ℤ</sub> × 10<sup>9</sup><sub>ℤ</sub> since if we are not doing proper scientific notation, this is the most illustrative way to write the multiplication in my opinion.